src/main/features/response/views/check-and-send.njk

```diff
@@ -64,6 +64,68 @@
     {% if statementOfMeansIsApplicable %}
       {{ tableStart('statementOfMeans', 'Your financial details') }}
       {{ row('Where are you living?	', draft.statementOfMeans.residence.residenceType, statementOfMeansPaths.residencePage.evaluateUri({ externalId: claim.externalId })) }}
+
+      {{ row('Children', '', statementOfMeansPaths.dependantsPage.evaluateUri({ externalId: claim.externalId }), false) }}
```

Comment:
what are those empty strings for that you are setting?
can you provide the variable name so that it is clear?